torch/nn/_functions/thnn/auto.py

```diff
@@ -333,4 +332,6 @@ def default_double_backwards_fn(ctx, *grad_params):
             _all_functions.append(backward_cls)
 
 
+from .auto_double_backwards import double_backwards_fns
```

Comment:
It should be possible to keep this import at the top, right?

Reply:
yes, will do.